lib/register.php

```diff
@@ -9,32 +9,19 @@
 	die( 'Silence is golden.' );
 }
 
-/**
- * Collect information about meta_boxes registered for the current post.
- *
- * Redirects to classic editor if a meta box is incompatible.
- *
- * @since 1.5.0
- * @deprecated 5.0.0 register_and_do_post_meta_boxes
- */
-function gutenberg_collect_meta_box_data() {
-	_deprecated_function( __FUNCTION__, '5.0.0', 'register_and_do_post_meta_boxes' );
-}
-
 /**
  * Return whether the post can be edited in Gutenberg and by the current user.
  *
  * @since 0.5.0
- * @deprecated 5.0.0 use_block_editor_for_post
+ * @deprecated 5.1.0 use_block_editor_for_post
  *
  * @param int|WP_Post $post Post ID or WP_Post object.
  * @return bool Whether the post can be edited with Gutenberg.
  */
 function gutenberg_can_edit_post( $post ) {
-	_deprecated_function( __FUNCTION__, '5.0.0', 'use_block_editor_for_post' );
+	_deprecated_function( __FUNCTION__, '5.1.0', 'use_block_editor_for_post' );
```

Comment:
Any particular reason, is this for the other issue? I'd personally be fine removing it now.

Reply:
> Any particular reason, is this for the other issue? I'd personally be fine removing it now.

These functions weren't actually deprecated in 5.0.0, they were deprecated in 5.1.0 . (They were originally targeted for but missed the release for 5.0.0 )